common/predictive-text/unit_tests/headless/edit-distance/distance-modeler.js

```diff
@@ -343,14 +343,17 @@ describe('Correction Distance Modeler', function() {
 
       //   set = set.value;
 
-      //   let entries = set[0].map(function(sequence) {
-      //     return sequence.map(value => value.key).join('');
+      //   let entries = set.map(function(result) {
+      //     return result.matchString;
```

Comment:
There have been a few code changes since this was originally written, so it needed some updating.